### DISCUSSION TITLE
[dupe] Contributor call: 19th of February 2022

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1851] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lucibytes on our Discord server.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:

Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: Saturday February the 19th, 2022 16:00 UTC, which is:
Los Angeles: 09:00
Chicago: 11:00
New York: 12:00
São Paulo: 14:00
London: 17:00
Paris: 18:00
Johannesburg: 19:00
Moscow: 20:00
Mumbai: 22:30
Hong Kong: 01:00 on Sunday
Tokyo: 02:00 on Sunday
Sydney: 04:00 on Sunday
Or check the bot in Discord! #meetings

### Agenda
- To be determined

> This discussion will be updated after the call with notes and to-dos.

**Comment by darigovresearch** [comment]
Would like to add the following ideas/discussion points for the next call if possible.  We will experiment with potential above solutions before the call and post closer to the time with references to aid the conversation

- Automate all the things?
  - Automating GitHub discussions to auto-fill default values such as labels, content field & title similar to pull request templates & issue templates for common discussion topics like these Contributor call discussion posts
  - Code to autogenerate time & date and common time zones given a date and UTC time that can be copy-pasted
  - Any other items which anyone wants to raise that they would like to see if it is possible to automate

**Comment by joostdecock** [comment, marked as answer]
Closing as a duplicate of #1833 (which predates this by 4 days)

Could you copy your suggestions there @darigovresearch ? We want to include them but should keep all info in one place.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
